backend/audit/models/viewflow.py

```diff
@@ -7,6 +7,25 @@
 logger = logging.getLogger(__name__)
 
 
+def sac_revert_from_submitted(sac):
+    """
+    Transitions the submission_state for a SingleAuditChecklist back
+    to "auditee_certified" so the user can re-address issues and submit.
+    This should only be executed via management command.
+    """
+
+    if sac.submission_status == STATUS.SUBMITTED:
+        flow = SingleAuditChecklistFlow(sac)
+
+        flow.transition_to_auditee_certified()
```

Comment:
Wrap:

```
with curation_tracking():
  sac.save()
```

Perhaps we should always wrap our `with` as close to the action as possible?

Reply:
https://stackoverflow.com/questions/28157929/how-to-safely-handle-an-exception-inside-a-context-manager

We might want to do a bit more reading to make sure we don't accidentally leave tracking on.

We should probably `disable_audit_tracking` somwhere at startup, as an "always-just-in-case"...